changelog.dd

```diff
@@ -28,6 +29,15 @@ $(LI $(LNAME2 TypeInfo.initializer, `TypeInfo.init` has been renamed to
         2.074.0 release, so that the type property `init` takes over.
     )
 )
+$(LI $(LNAME2 AA.clear, Associative array `clear` method to remove all
+    elements has been added.)
+
+    $(P One can now use `aa.clear()` to remove all elements from an
+        associative array. This allows one to remove all elements from all
+        references to the same array (setting to `null` just cleared the
+        reference, not removed the elements).
+    )
```

Comment:
I think the second part is a bit hard to understand and would be better illustrates by a concise code example.
```d
auto aa = ["first": "entry", " second": "entry"];
aa.clear(); // still retains it's current capacity for faster inserts
assert(aa.length == 0);
aa.rehash(); // explicitly reset the capacity
```
```d
auto aa1 = ["first": "entry", " second": "entry"];
auto aa2 = aa1; // reference to the same AA
aa1.clear(); // clear the referenced AA
assert(aa2.length == 0);
```